experimental/dds/tree/src/migration-shim/migrationDeltaHandler.ts

```diff
@@ -119,16 +119,18 @@ export class MigrationShimDeltaHandler implements IShimDeltaHandler {
 		return this.treeDeltaHandler.reSubmit(contents, localOpMetadata);
 	}
 
-	public applyStashedOp(contents: unknown): unknown {
+	public applyStashedOp(contents: unknown): void {
 		const opContents = contents as IOpContents;
 		if (this.isInV1StateAndIsBarrierOp(opContents)) {
-			return undefined;
+			this.submitLocalMessage(opContents);
+			return;
 		}
 
 		if (this.shouldDropOp(opContents)) {
-			return undefined;
+			this.submitLocalMessage(opContents);
```

Comment:
i think this should probably not submit anything, @tyler-cai-microsoft to confirm

Reply:
ah. always takes me a minute to page in that old flow has applyStashedOp called first, then the regular resubmit flow afterward.

I'm not actually sure when this conditional is meaningfully hit today. DDS should be in the same state as when it originally submitted the op, so opContents should not be 'in the wrong format' (i.e. using the format from the other side of a barrier op). I think it might be better converted to an assert or something. Obviously not for this PR though.

Reply:
It should not submit anything. That would be best. Specifically, if I go offline and the tree goes from v1 to v2, then we want any v1 ops to be dropped. It would be optimal, and would not block any future scenarios at which we wish to get rid of the `MigrationDeltaHandler` and `ShimDeltaHandler` at some point in the lifecycle of a document after migration.

Reply:
If you look at resubmit, resubmit does not send any stashed v1 ops 
```
if (this.shouldDropOp(opContents)) {
    return;
}

Reply:
The customer scenario here is that the customer using this logic will not go offline, at least, they don't have any offline functionality for their current documents that are in the v1 state.

Reply:
for apply stashed ops we don't know if the op was already sent or not, so apply stashed op needs to assume it could've been sent, it will have a chance to not send if it wasn't already sent in resubmit.

Reply:
For apply stashed ops, we are rebasing those ops right?

Reply:
they will get rebased only if they are not received as remote ops. if they are not received as remote ops, they go through the normal resubmit flow to get rebased and submitted

Reply:
I just realized, this is probably best to be an assert, this shouldn't have happened

Reply:
yeah converting to an assert seems right here. but not for this PR :). Tyler would you mind following up on that whenever appropriate?

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/7163